packages/@internationalized/number/src/NumberFormatter.ts

```diff
@@ -124,7 +124,7 @@ function getCachedNumberFormatter(locale: string, options: NumberFormatOptions =
 }
 
 /** @private - exported for tests */
-export function numberFormatSignDisplayPolyfill(numberFormat: Intl.NumberFormat, signDisplay: 'always' | 'exceptZero' | 'auto' | 'never', num: number) {
+export function numberFormatSignDisplayPolyfill(numberFormat: Intl.NumberFormat, signDisplay: string, num: number) {
```

Comment:
Why did this get less specific?

Reply:
string is the type that TypeScript has, we'd defined it ourselves earlier, but this fits with what its actual type is https://github.com/microsoft/TypeScript/search?q=signDisplay